docs/sources/installation/helm/configure-storage/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Configure storage
+menuTitle: Configure storage 
+description: Configure Loki storage
+aliases:
+  - /docs/writers-toolkit/latest/templates/task-template
+weight: 100
+keywords:
+  - object store
+  - filesystem
+  - minio
+---
+
+# Configure Loki's storage
+
+The [scalable](../install-scalable/) installation requires a managed object store such as AWS S3 or Google Cloud Storage or a self-hosted store such as Minio. The [single binary](../install-monolithic/) installation can only use the filesystem for storage.
+
+**To use a managed object store:**
+
+1. Set the `type` of `storage` in `values.yaml` to `gcs` or `s3`.
+
+2. Configure the storage client under `storage.gcs` or `storage.s3`.
+
+
+**To install Minio alongside Loki:**
+
+1. Change the configuration in `values.yaml`:
+
+    - Enable Minio
+
+    ```yaml
+    minio:
+      enabled: true
+    ```
+
+    - Remove the `storage` section. It will be configured automatically to use the Minio installation.
```

Comment:
They start from an installation but I don't know if there's a storage there. This is stitch together. I need to take a closer look.

Reply:
so this is assuming an existing installation that they are moving to minio? if so that makes sense, but maybe we reword to "remove storage section if present" or something?

Reply:
You are right. If they start from scratch there's nothing to remove.